### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by westnight** [sirius-ai/MobileFaceNet_TF#1]
hey，you have done good job! But in my training process, the loss increase to about ±18 when an epoch ends and another begin, can you tell me why?

**Comment by xsr-ai** [comment]
what best accuracy are you reach? maybe you should minimize the learning rate.

**Comment by sirius-ai** [comment]
604e36b
close, when need reopen.

**Comment by flyduck** [comment]
@sirius-ai 您好，我看了您的代码，在计算损失函数那个地方，有个mm =sinm × m，这里是为什么需要在sinm上乘以一个m啊？后面注释写的是解决issue1的～
谢谢～
